ContextDesktop/src/main/java/com/github/migi_1/Context/model/entity/Camera.java

```diff
@@ -0,0 +1,30 @@
+package com.github.migi_1.Context.model.entity;
+
+import com.jme3.scene.Node;
+import com.jme3.scene.Spatial;
+
+import jmevr.app.VRApplication;
+
+/**
+ * Class that handles the debugging camera.
+ * It is treated as a spatial in order to move it around actively.
+ * The flyCam is unable to be used in this application because it is an VR application.
+ * This is the alterative.
+ * @author Marcel
+ *
+ */
+public class Camera extends Entity {
+
+    /**
+     * Displays the view of this camera on the screen.
+     */
+    public void makeObserver() {
+    	VRApplication.setObserver(this.getModel());
+    }
```

Comment:
I would say not, because you will need to create a new one each time you enter the environment state. If you use a Singleton you would have to reset the object.